.ci/config/config.compression+ssl.json

```diff
@@ -4,7 +4,11 @@
       "PasswordlessUser": "no_password",
       "SecondaryDatabase": "testdb2",
       "SupportedFeatures": "Json,StoredProcedures,Sha256Password,LargePackets",
+	  
+      "TestData": "../../../../TestData",
       "MySqlBulkLoaderLocalCsvFile": "%TESTDATA%/LoadData_UTF8_BOM_Unix.CSV",
-      "MySqlBulkLoaderLocalTsvFile": "%TESTDATA%/LoadData_UTF8_BOM_Unix.TSV"
+      "MySqlBulkLoaderLocalTsvFile": "%TESTDATA%/LoadData_UTF8_BOM_Unix.TSV",
+	  
+	  "CertificatesPath": "..\\..\\..\\..\\..\\.ci\\server\\certs"
```

Comment:
Sort of, I figured safer to just put the logic in AppConfig to normalize it no matter what user inputs.

Should be ok now.